playbooks/roles/logstash/files/template_logstash.json

```diff
@@ -1,36 +0,0 @@
-{
```

Comment:
we do, I just moved it over to the elasticsearch role. you can technically define the template in either one, but 
1) the way the two interact, putting the template in elasticsearch picks it up by default
2) developing on ELK stack as we have it means spending more time on elasticsearch and almost none on logstash, so it makes more sense for a developer to be able to look at it there